src/analytics/lambdas/load-data-workflow/has-more-job-new.ts

```diff
@@ -22,24 +22,26 @@ const ODS_EVENT_BUCKET = process.env.ODS_EVENT_BUCKET!;
 const ODS_EVENT_BUCKET_PREFIX = process.env.ODS_EVENT_BUCKET_PREFIX!;
 const DYNAMODB_TABLE_NAME = process.env.DYNAMODB_TABLE_NAME!;
 const DYNAMODB_TABLE_INDEX_NAME = process.env.DYNAMODB_TABLE_INDEX_NAME!;
-const REDSHIFT_ODS_TABLE_NAME = process.env.REDSHIFT_ODS_TABLE_NAME!;
 
 export const handler = async (_: any, context: Context) => {
```

Comment:
rename `_` to `event`, and use the type

same for other handlers

Reply:
Refactored them.